src/Compilers/Core/MSBuildTaskTests/DotNetSdkTests.cs

```diff
@@ -324,7 +324,7 @@ public void InitializeSourceRootMappedPathsReturnsSourceMap(bool deterministicSo
             VerifyValues(
                 customProps: $@"
 <ItemGroup>
-<ProjectReference Include=""Project2.csproj"" Targets=""InitializeSourceRootMappedPaths"" OutputItemType=""ReferencedProjectSourceRoots"" ReferenceOutputAssembly=""false"" />
+    <ProjectReference Include=""Project2.csproj"" Targets=""InitializeSourceRootMappedPaths"" OutputItemType=""ReferencedProjectSourceRoots"" ReferenceOutputAssembly=""false"" />
```

Comment:
❔ Why did this change? The expected indent here is 2 spaces.

```suggestion
  <ProjectReference Include=""Project2.csproj"" Targets=""InitializeSourceRootMappedPaths"" OutputItemType=""ReferencedProjectSourceRoots"" ReferenceOutputAssembly=""false"" />
```

Reply:
It didn't really seem consistent, but I am happy to update it.